edbterraform/data/terraform/aws/modules/specification/variables.tf

```diff
@@ -83,12 +83,13 @@ variable "spec" {
         encrypted = optional(bool)
       })
       additional_volumes = optional(list(object({
-        mount_point = string
-        size_gb     = number
-        iops        = optional(number)
-        type        = string
-        encrypted   = optional(bool)
-        filesystem  = optional(string)
+        mount_point        = string
+        size_gb            = number
+        iops               = optional(number)
+        type               = string
+        encrypted          = optional(bool)
+        filesystem         = optional(string)
+        filesystemmountopt = optional(string)
```

Comment:
Update variable name and update type with a default: `optional(list(string))`

Reply:
ok